.eslintrc

```diff
@@ -1,33 +1,24 @@
 {
-  "rules": {
-    "indent": [2, 2, {
-      "VariableDeclarator": {
-        "var": 2,
-        "let": 2,
-        "const": 3
-      }
-    }],
-    "no-empty": [2],
-    "no-console": [0],
-    "comma-dangle": [2],
-    "no-unused-vars": [0],
-    "linebreak-style": [2, "unix"],
-    "semi": [2, "always"]
-  },
+  "parser": "babel-eslint",
+  "extends": "./node_modules/mofo-style/.eslintrc.yaml",
   "env": {
+    "node": true,
     "commonjs": true,
     "es6": true,
     "browser": true
   },
-  "extends": "eslint:recommended",
   "parserOptions": {
+    "ecmaVersion": 6,
     "ecmaFeatures": {
-      "es6": true,
-      "jsx": true,
-      "experimentalObjectRestSpread": true
+      "jsx": true
     }
   },
+  "rules": {
+    "no-console": 1,
+    "react/jsx-uses-react": 1,
+    "react/jsx-uses-vars": 1
```

Comment:
So for testing purposes I required a `<Sidebar>` component and never used it later in the file. Turning the `react/jsx-uses-vars` linting rule on (doesn't matter if it's set to `1` or `2`) gives us the correct linting result:
<img width="571" alt="screen shot 2016-02-23 at 11 10 07 pm" src="https://cloud.githubusercontent.com/assets/2896608/13278203/98a68a96-da83-11e5-9db9-030af03c6913.png">

Turning that rule off will give us the wrong flags as seen on [my previous comment](https://github.com/Pomax/ReactRoutedExample/pull/3#discussion_r53673512)